web/controllers/user_controller.ex

```diff
@@ -11,6 +11,22 @@ defmodule CoursePlanner.UserController do
     render(conn, "index.html", users: Users.all())
   end
 
+  def show(%{assigns: %{current_user: %{role: role}}} = conn, %{"id" => id}) do
+    if role === "Coordinator" do
```

Comment:
Every user had access to their own show page before, that's what I tried to stop. We won't redirect them to the user page ever